homeassistant/components/frontier_silicon/__init__.py

```diff
@@ -1 +1,38 @@
-"""The frontier_silicon component."""
+"""The Frontier Silicon NEW integration."""
+from __future__ import annotations
+
+from afsapi import AFSAPI
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .const import CONF_PIN, CONF_USE_SESSION, CONF_WEBFSAPI_URL, DOMAIN
+
+PLATFORMS: list[str] = ["media_player"]
```

Comment:
```suggestion
PLATFORMS = [Platform.MEDIA_PLAYER]
```

+ `from homeassistant.const import Platform`